layouts/contact/list.html

```diff
@@ -3,55 +3,279 @@
+      formData.append('email', document.getElementById('email').value);
+      formData.append('message', document.getElementById('message').value);
+      formData.append('originUrl', document.getElementById('originUrl').value);
+      formData.append('origin', 'general_contact');
+      
+      // Session data
+      const currentTime = Date.now();
+      const duration = Math.round((currentTime - sessionStart) / 1000);
+      formData.append('session_id', sessionStart);
+      formData.append('session_duration', duration);
+      
+      // Additional fields
+      formData.append('website', document.getElementById('website').value);
+      formData.append('phone_number', document.getElementById('phone_number').value);
+      formData.append('company', document.getElementById('company').value);
+      
+      // reCAPTCHA data - commented out to test anti-spam
+      /*
+      var recaptchaResponse = $("#g-recaptcha-response").val();
+      if (recaptchaResponse) {
+        formData.append('g-recaptcha-response', recaptchaResponse);
+      }
+      */
+
+      // Submit to Google Apps Script
+      fetch('https://script.google.com/macros/s/AKfycbzDm3HImX8rx052Ooe888T702iuAYNosV4zO3mgSgucQ1IKCI2hhP_EMhFI82lMqDDT/exec', {
+        method: 'POST',
+        body: formData
+      })
+      .then(response => response.json())
+      .then(data => {
+        if (data.success) {
+          console.log('Form submitted successfully:', data.message);
+          showSuccess(data.message);
+        } else {
+          console.error('Form submission failed:', data.message);
+          showError(data.message);
+        }
+      })
+      .catch(error => {
+        console.error('Form submission error:', error);
+        showError();
+      })
+      .finally(() => {
+        // Reset button state
+        submitBtn.disabled = false;
+        btnText.textContent = 'Send';
+        spinner.classList.add('d-none');
+      });
+    });
+
+    // Also prevent form submission if user presses Enter
+    form.addEventListener('submit', function(event) {
       event.preventDefault();
-      alert("Please tick 'I'm not a robot' above the 'Submit the form' button.");
+      event.stopPropagation();
+      return false;
+    });
+
+    function validateSubmission() {
+      // Check additional fields
+      const website = document.getElementById('website').value;
+      const phone = document.getElementById('phone_number').value;
+      const company = document.getElementById('company').value;
+      
+      if (website || phone || company) {
+        console.log('Additional fields validation failed');
+        return false;
+      }
+      
+      // Check session timing
+      const currentTime = Date.now();
+      const elapsed = (currentTime - sessionStart) / 1000;
+      
+      if (elapsed < 5) {
+        console.log('Session too short:', elapsed);
+        return false;
+      }
+      
+      if (elapsed > 3600) {
```

Comment:
Wondering if we need this ? Only (or mostly) a human user will take longer than 60 minutes to submit the form, that too I can only presume a scenario where they get distracted with other stuff after open the contact page and later comeback to the form, they fill and submit, then it would just fail without specific reason.

Reply:
It's less mandatory than `< 5 seconds` for sure, I can remove it